pyomo/solvers/plugins/solvers/SCIPAMPL.py

```diff
@@ -216,70 +277,108 @@ def _postsolve(self):
+        #                                                 #    but having return a feasible solution
+        
         elif results.solver.message == "time limit reached":
             results.solver.status = \
-                SolverStatus.aborted
+                SolverStatus.ok
             results.solver.termination_condition = \
                 TerminationCondition.maxTimeLimit
             if len(results.solution) > 0:
                 results.solution(0).status = \
                     SolutionStatus.stoppedByLimit
+                    
+        # # OK
+        # other='other'                                   # Other, uncategorized normal termination
+        
         elif results.solver.message == "memory limit reached":
             results.solver.status = \
-                SolverStatus.aborted
+                SolverStatus.ok
             results.solver.termination_condition = \
                 TerminationCondition.other
             if len(results.solution) > 0:
                 results.solution(0).status = \
                     SolutionStatus.stoppedByLimit
+                    
+        # # OK
+        # other='other'                                   # Other, uncategorized normal termination
+                 
         elif results.solver.message == "gap limit reached":
             results.solver.status = \
-                SolverStatus.aborted
+                SolverStatus.ok
             results.solver.termination_condition = \
                 TerminationCondition.other
             if len(results.solution) > 0:
                 results.solution(0).status = \
                     SolutionStatus.stoppedByLimit
+                    
+        # # OK
+        # other='other'                                   # Other, uncategorized normal termination
+                    
         elif results.solver.message == "solution limit reached":
             results.solver.status = \
-                SolverStatus.aborted
+                SolverStatus.ok
             results.solver.termination_condition = \
                 TerminationCondition.other
             if len(results.solution) > 0:
                 results.solution(0).status = \
                     SolutionStatus.stoppedByLimit
+                    
+        # # OK
+        # other='other'                                   # Other, uncategorized normal termination
+        
         elif results.solver.message == "solution improvement limit reached":
             results.solver.status = \
-                SolverStatus.aborted
+                SolverStatus.ok
             results.solver.termination_condition = \
                 TerminationCondition.other
             if len(results.solution) > 0:
                 results.solution(0).status = \
                     SolutionStatus.stoppedByLimit
+                    
+        # # OK
+        # optimal='optimal'                               # Found an optimal solution
```

Comment:
I appreciate these comments being inserted, but can they please be cleaned up a bit? I.e., one comment per line?